squarelet/users/serializers.py

```diff
@@ -154,6 +155,24 @@ def __init__(self, *args, **kwargs):
                 self.fields["username"].read_only = True
 
 
+class PressPassUserInvitationsSerializer(FlexFieldsModelSerializer):
+    organization = serializers.SlugRelatedField(slug_field="uuid", read_only=True)
+
+    class Meta:
+        model = Invitation
+        fields = (
+            "uuid",
+            "user",
```

Comment:
I would remove user - since this is only viewable under a given user, it should be unnecessary (similarly, the invitations nested under organization do not include it in its fields)